README.md

```diff
@@ -203,8 +230,37 @@ To include all the files in a directory, use the `*` wild card (eg. `/var/log/*`
 
 Additionally, you can add the `--no_service` to omit the step of automatically creating `monasca-log-agent.service` in `/etc/systemd/system/`
 
-### Monasca-ui
+### Uninstalling Logs agent
+In the following description, 
+[target_dir] is the target directory specified at the time of running log-agent-[version].run
+
+stop the service:
+```
+systemctl stop monasca-log-agent
+```
+
+delete obsolete files: 
```

Comment:
`/etc/systemd/system/monasca-log-agent.service` is also created by Log agent installer (and should be removed I think).

Reply:
It's removed in line 247.